acpi/src/platform/mod.rs

```diff
@@ -132,3 +132,55 @@ where
         Ok(PlatformInfo { power_profile, interrupt_model, processor_info, pm_timer })
     }
 }
+
+/// Wakes up Application Processors (APs) using the Multiprocessor Wakeup Mailbox mechanism.
+///
+/// This function writes to a specific memory-mapped mailbox to signal APs to wake up and start
+/// executing from a given wakeup vector. It waits for the APs to acknowledge the wakeup command
+/// by monitoring the mailbox command field until it is reset to `Noop`.
```

Comment:
We can merge without as people should be reading the ACPI specification themselves too, but some documentation on what the code at the wakeup vector should expect (e.g. a single page identity-mapped, and the state of the processor) might be nice.